src/Microsoft.DotNet.Cli.Utils/Command.cs

```diff
@@ -28,6 +28,7 @@ private Command(CommandSpec commandSpec)
                 RedirectStandardError = true,
                 RedirectStandardOutput = true
             };
+            psi.Environment[CommandContext.Variables.AnsiPassThru] = bool.TrueString;
```

Comment:
I think there are situations where we might want to use command without being forced to set this value. Testing is the first thing that comes to mind, but also this is intended to be a general purpose Command Class for use by CLI.

I prefer the more explicit method of declaring exactly what environment variables you want set with each command.